kernel/data-pipeline/api/src/main/java/org/apache/shardingsphere/data/pipeline/api/pojo/TableBasedPipelineJobInfo.java

```diff
@@ -32,4 +32,10 @@ public final class TableBasedPipelineJobInfo implements PipelineJobInfo {
     private final String databaseName;
     
     private final String table;
+    
+    public TableBasedPipelineJobInfo(final PipelineJobMetaData jobMetaData, final String table) {
+        this.jobMetaData = jobMetaData;
+        this.table = table;
+        this.databaseName = null;
```

Comment:
1, `this.databaseName` could be `databaseName`.

2, Should `databaseName = null;` put before `this.table = table;`?